web/Controllers/AdminController.cs

```diff
@@ -74,12 +74,27 @@ public IActionResult CancelAdminAccessRequest()
 
     [Authorize(Roles="Admin")]
     [HttpGet]
-    public async Task<IActionResult> AdminList()
+    [Route("AdminList")]
+    public async Task<IActionResult> GetAdminList()
     {
         AdminListModel adminListModel = new AdminListModel();
+        
         var adminLists = await _adminService.GetAdminLists();
+        
         adminListModel.CurrentAdmins = adminLists.CurrentAdmins;
         adminListModel.PendingAdmins = adminLists.PendingAdmins;
-        return View(adminListModel);
+        
+        return View("AdminList", adminListModel);
+    }
+
+    [Authorize(Roles="Admin")]
+    [HttpPost]
+    public async Task<ActionResult> AcceptAdminRequest(AdminListModel userToAcceptList, int userToAcceptId)

```

Comment:
If this is async, the admin service methods you call should be too, then you can await them